core/src/main/scala/com/raphtory/internals/storage/pojograph/entities/external/vertex/PojoReversedView.scala

```diff
@@ -9,6 +9,7 @@ import scala.collection.View
 private[pojograph] class PojoReversedVertexView[T](override val vertex: PojoConcreteVertexBase[T])
         extends PojoLocalVertexViewBase(vertex) {
 
+  override def getStateSet(): List[String] = ???
```

Comment:
Implement this by pointing at the underlying vertex (implementation should go in `PojoLocalVertexViewBase`